### DISCUSSION TITLE
Welcome to FXGL Discussions!

### DISCUSSION THREAD

**Discussion by AlmasB** [AlmasB/FXGL#1011] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask FXGL related questions.
  * Share your projects built with FXGL.
  * Propose new features and ideas.
  * Engage with other community members.
  * Welcome others and are open-minded, and follow our [Code of Conduct](https://github.com/AlmasB/FXGL/blob/dev/CODE_OF_CONDUCT.md). Remember that this is a community we
  build together 💪.

Previously, all of above was done via the [Gitter Community Chat](https://gitter.im/AlmasB/FXGL). The chat will be kept active for the time being with the long-term plan to fully migrate to GitHub Discussions.

To get started, comment below with an introduction of yourself and tell us about your interest in this project!

**Comment by bgueye1** [comment]


**Reply by AlmasB** [reply]
Hi, thanks for your interest. If you have errors in the build process, please double check with [development](https://github.com/AlmasB/FXGL/blob/dev/CONTRIBUTING.md#development). If the errors are not covered there, feel free to post them here or on [Gitter](https://gitter.im/AlmasB/FXGL).
